app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
can we avoid the extra require? loading in an entire templating engine for this feels kinda overkill. how about something dead simple like:

THE-HUE --> BLUE-GREY
THE_HUE --> BLUE_GREY
the-hue --> blue-grey